### ISSUE TITLE
pip install --no-binary fails because Cython command: cython --version is failing. Only pip >= 19

### ISSUE THREAD

**Issue by atilag** [scikit-build/scikit-build#390]
Hi!
When I try to install my project via:
```
$ pip install -v ~/my-project --no-binary my-project
```
I'm getting this error:
```
CMake Error at /tmp/pip-build-env-n1bmy3uo/overlay/lib/python3.7/site-packages/skbuild/resources/cmake/FindCython.cmake:64 (message):
      Command "/home/jgomez/anaconda3/envs/test1/bin/cython;--version" failed
      with output:

      Traceback (most recent call last):

        File "/home/jgomez/anaconda3/envs/test1/bin/cython", line 6, in <module>
          from Cython.Compiler.Main import setuptools_main

      ModuleNotFoundError: No module named 'Cython'

    Call Stack (most recent call first):
      qiskit/providers/aer/backends/wrappers/CMakeLists.txt:4 (find_package)
```
As the output shows, I have already installed cython in my virtual env, but the command seems to be malformed?: `/home/jgomez/anaconda3/envs/test1/bin/cython;--version` not quite sure this is the actual error, because the following output error message doesn't match with any possible output from this command, so not sure what's going on here.

Everything works fine when I just build directly from the project sources with `bdist_wheel`.

**Comment by atilag** [comment]
Ok, this must be related to some change in pip version 19, because everything works fine with pip 18.

**Comment by atilag** [comment]
Ok, after some debugging, I have find out that the cython command is ok. The semicolon is expected per CMake specifications. It's the execution of the `cython` command the one that is failing, don't know why yet though. Gonna change the title of the issue for the sake of correctness.

**Comment by atilag** [comment]
This seems to be an specific pip issue, so I'm closing this.

**Comment by jcfr** [comment]
Thanks for the report @atilag 

I also plan to investigate the problem, in the mean time I am re-opening.

**Comment by atilag** [comment]
I think that this is the problem: 
https://github.com/pypa/pip/issues/6222

I have filed a new issue though:
https://github.com/pypa/pip/issues/6360

**Comment by atilag** [comment]
Ok, this is fixed as of `pip 19.3.1` :)
